test/image.bats

```diff
@@ -516,3 +516,32 @@ EOF
 
 	run crictl run "$TESTDIR"/memory.json "$TESTDATA"/sandbox_config.json
 }
+
+@test "run container having memory limit below default 12 MiB with empty container_min_memory" {
+	cat << EOF > "$CRIO_CONFIG_DIR/99-mem.conf"
+[crio.runtime]
+default_runtime = "mem"
+[crio.runtime.runtimes.mem]
+runtime_path = "$RUNTIME_BINARY_PATH"
+EOF
+	start_crio
+
```

Comment:
Can we check if the log file contains the line about the `container_min_memory` default value?
Something like:
```sh
wait_for_log "Runtime handler \"runc\" container minimum memory set to 12582912 bytes"
```

This can supplement the test with the necessary verification. 🤔